### DISCUSSION TITLE
horizon line

### DISCUSSION THREAD

**Discussion by tzmosh** [Stellarium/stellarium#2581] [Ideas]
The horizon line is indicated in the stellarium at a distance of 90 degrees from the zenith.
Also celestial events, for example, a sunrise is excellent as a transit time over the horizon line.
Those who stand on the top of a high mountain manage to see even lower parts, from the horizon line that is marked in the sky.
Is it possible to change the horizon line to the limit of the visible sky.
And correspondingly, the heavenly events will also change.
Thanks in advance.

**Comment by gzotti** [comment]
https://github.com/Stellarium/stellarium/wiki/Common-Problems-for-the-current-version#when-i-raise-the-altitude-of-my-location-why-does-the-landscape-not-move-down

**Comment by tzmosh** [comment]
I am referring to the change in height of the entire horizon regardless of any features on the horizon.
Like a mountain on the seashore, when the horizon looks like a straight line.
Changing the panorama does not change the calculated astronomical events, such as the time of sunrise, for example.

**Reply by gzotti** [reply]
Sunrise is always computed with respect to the mathematical horizon which is always 90° from the zenith. At some point we will hopefully be able to compute times with respect to the currently set *landscape*. If you create a correctly adjusted *landscape* for your mountain site, you will be able to see things like the squashed sun below the mathematical horizon. However, in these altitudes the atmosphere is so variable that any prognosis or computation should not be trusted. For technical reasons, as discussed in the User Guide, the refraction model breaks down somewhere below the mathematical horizon.

**Comment by tzmosh** [comment]
As always your answers are clear and helpful.
Thank you.
